tests/storage/mpathconf_test.py

```diff
@@ -224,3 +224,157 @@ def test_mpathd_parser():
             Section('entry', [])
         ]),
     ]
+
+
+def test_mpathd_ufn_enabled_single():
+    fake_conf = [
+        Section('default', [
+            Pair('key', 'value'),
+            Pair('user_friendly_names', 'yes'),
+        ]),
+        Section('devices', [
+            Section('device', [
+                Pair('user_friendly_names', 'no'),
+                Pair('key', 'value'),
+            ]),
+            Section('device', [
+                Pair('key', 'value')
+            ])
+        ]),
+    ]
+    issues = mpathconf._check_conf(fake_conf)
+    assert issues == [
+        Section('default', [
+            Pair('key', 'value'),
+            Pair('user_friendly_names', 'yes'),
+        ])
+    ]
+
+
+def test_mpathd_ufn_enabled_multiple():
+    fake_conf = [
+        Section('default', [
+            Pair('key', 'value'),
+            Pair('user_friendly_names', 'yes'),
+        ]),
+        Section('devices', [
+            Section('device', [
+                Pair('user_friendly_names', 'yes'),
```

Comment:
and this cover the first pair case. If we add another device or multipath to the test data using middle pair, we cover the case.